app/query/caller.go

```diff
@@ -362,3 +365,22 @@ func isErrWalletNotLoaded(r *jsonrpc.RPCResponse) bool {
 func isErrWalletAlreadyLoaded(r *jsonrpc.RPCResponse) bool {
 	return r.Error != nil && errors.Is(lbrynet.NewWalletError(0, errors.Err(r.Error.Message)), lbrynet.ErrWalletAlreadyLoaded)
 }
+
+// Makes a shallow copy of a map, cutting the size of the lists inside it
```

Comment:
Function comment should starts with function name [as per the convention](https://go.dev/blog/godoc).